scenario_configs/so2sat_sar_masked_pgd_defended.json

```diff
@@ -39,37 +37,37 @@
                 1.0
             ],
             "mins": [
-                -1.0,
-                -1.0,
-                -1.0,
-                -1.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0
+                -0.22,
```

Comment:
how were the `"mins"` and `"ranges"` values determined for so2sat configs? I do recall conversation that the previous bounds were loose compared to the actual distribution of benign data

Reply:
@lcadalzo I found the min/max by cycling through each channel in the test data. The ranges = max - min.